packages/jellyfish-transaction-builder/src/txn/txn_builder_masternode.ts

```diff
@@ -41,4 +48,90 @@ export class TxnBuilderMasternode extends P2WPKHTxnBuilder {
       changeScript
     )
   }
+
+  /**
+   * Build update masternode transaction
+   *
+   * @param {UpdateMasternode} updateMasternode transaction to create
+   * @param {Script} changeScript to send unspent to after deducting the (converted + fees)
+   * @param {Object} [collateralInfo] needed when updating owner address
+   * @param {RawTransaction} collateralInfo.rawCollateralTx to get relevant data needed for collateral vin and vout
+   * @param {Script} collateralInfo.newOwnerScript for new owner address
+   * @return {Promise<TransactionSegWit>}
+   */
+  async update (
+    updateMasternode: UpdateMasternode,
+    changeScript: Script,
+    collateralInfo?: {
```

Comment:
```suggestion
    collateral?: {
```

IMO, we could just call it `collateral` since all datatypes are analogous to `info` being passed.